libvast/src/system/node.cpp

```diff
@@ -332,14 +335,23 @@ node_state::spawn_command(const invocation& inv,
       VAST_DEBUG(this_node, "auto-generated new label:", label);
     }
   }
+  VAST_DEBUG(this_node, "spawns a", comp_name, "with the label", label);
+  auto spawn_inv = inv;
```

Comment:
Should this be a copy or an alias?

Reply:
A copy is required, because the function signature is mandated by the factory, and that requires `inv` to be const.